src/components/AppNavBar/AppNavBarPresentation.js

```diff
@@ -166,6 +167,15 @@ function AppNavBarPresentation({ isHome = false, login, logout }) {
               {getIcon('QuestionCircleSolid', 'fill-current h-full w-6 mr-2')}
               Support
             </Link>
+            {/* Search */}
+            {!isHome && (
+              <div
+                id="AppNavSearch"
+                className="inline-flex items-center px-8 py-2"
```

Comment:
The spacing of the search box looks fine when a user is logged in, however it looks a bit squished when the user is logged out and the icon is replaced by the "Sign in / Register" menu item. If this spacing needs to be adjusted please let me know

Reply:
Try removing `md:w-1/2 2xl:w-1/4` from the ul classes on line 162. We're already justifying the links to the end so constricting the width isn't really necessary and should allow more space for the search.

Reply:
Perfect, that fixed the spacing, thanks!